source/sass/global.scss

```diff
@@ -91,3 +91,7 @@ button:hover {
     color: transparentize($white, 0.5);
   }
 }
+
+.hidden {
```

Comment:
Bootstrap should have a class for this already

Reply:
it does, `d-none`, which is a terrible name. I'll use it, but let the record show that the only right name for this CSS is `hidden`.